reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,325 @@
+.. index::
+   single: Forms; Fields; DateIntervalType
+
+DateIntervalType Field Type
```

Comment:
We don't usually include the last word `Type` - so just `DateIntervalType Field`